### DISCUSSION TITLE
Avast reports that react-three_drei.js is infected with JS:Prontexi-Z[Trj].

### DISCUSSION THREAD

**Discussion by lowpolypanda** [pmndrs/drei#2222] [Q&A]
I received an Avast notification stating that react-three_drei.js is infected with JS:Prontexi-Z[Trj] and it has been sent to Quarantine. Could this be a false positive?

**Thank you for your help!**

**Comment by JamesMasino** [comment]
This seems to be a known false flag on Avast and several other virus scanners, I opened an issue thread for this: https://github.com/pmndrs/drei/issues/2239

Nobody has tracked down exactly what's causing it, and Avast hasn't offered much information about it. I personally think it's flagging an older threat pattern that's no longer an issue, I've developed with the drei library extensively and haven't had any problems. I have the file exempt from being quarantined.
